gns3/registry/config.py

```diff
@@ -236,7 +238,7 @@ def _add_qemu_config(self, new_config, appliance_config):
 
         new_config["kernel_command_line"] = appliance_config["qemu"].get("kernel_command_line", "")
 
-        new_config["qemu_path"] = "qemu-system-{}".format(appliance_config["qemu"]["arch"])
```

Comment:
Yeah you are right a story for another time :P

On Mon, Jan 4, 2016 at 7:40 PM Vasil Rangelov <notifications@github.com>
wrote:

> In gns3/registry/config.py
> <https://github.com/GNS3/gns3-gui/pull/904#discussion_r48763292>:
>
> > @@ -236,7 +238,7 @@ def _add_qemu_config(self, new_config, appliance_config):
> >
> >          new_config["kernel_command_line"] = appliance_config["qemu"].get("kernel_command_line", "")
> >
> > -        new_config["qemu_path"] = "qemu-system-{}".format(appliance_config["qemu"]["arch"])
>
> Oh wait, you mean the .gns3 file portability? Yeah, about that... that has
> never truly been there yet. To get that level of portability, the arch
> needs to be part of the topology, and in a separate place (perhaps the Qemu
> preferences window, or perhaps in a separate tab inside there) define which
> binary is for which arch, with the defaults being inferred in the current
> fashion.
>
> (Again, "a story for another time"...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GNS3/gns3-gui/pull/904/files#r48763292>.
>